### DISCUSSION TITLE
Is it possible to increase performance of our nuxt app?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#27631] [Questions]
So, we are planning to create a new web app. We are about to use Nuxt when the team discovered this chart. And we are in the process of choosing what is best. Now a days we are more into performance. Is their a way to increase Nuxt app performance?

based on this graph nuxt is the last one, which is making us thinking about using astro instead.
![image](https://github.com/nuxt/nuxt/assets/31676163/8810d8e1-f329-4914-8800-72ed5d73a947)

**Comment by jrmybtlr** [comment]
There are endless options for the performance and optimisation of Nuxt (and Next). Pre-rendering, server components, server pages, etc, are the most accessible core options in Nuxt to create incredible performance and CWV metrics.

Astro and Islands generally stream server-rendered HTML by default, so the comparison is only valid when you have a side-by-side test of the same site under the same conditions in each framework. Their advantage is the base configuration.

They use aggregated data above, which always leads to losing vital information. It doesn't represent the best or worst of any of the platforms listed. It looks great for a chart, but!
